### DISCUSSION TITLE
Uploading file larger than 1mb fails

### DISCUSSION THREAD

**Discussion by WeatherAnchor** [pockethost/pockethost#412] [Q&A]
I am using Pockethost.io and all the things are working really well. I setup a collection with a file field and its maximum file size is about 50mb right now. I have tried uploading from the admin dashboard and tried uploading using POST request. But still failing to upload and giving a response of 413 Request Entity Too Large.

Here is the related issue. **https://github.com/pocketbase/pocketbase/discussions/883**

**Comment by benallfree** [comment]
Ok interesting, let me think about this one a bit and come up with a more reasonable limit. We don't have storage quotas implemented yet. Marking for v0.6

**Comment by benallfree** [comment]
moved to roadmap

**Comment by eevleevs** [comment]
Still unable to upload large files, can you please inform if the limit has been raised?

**Comment by benallfree** [comment]
@eevleevs How large are you trying to upload?

**Comment by eevleevs** [comment]
About 25 MB. I can upload it from the admin UI, but not through the JS client.

**Reply by benallfree** [reply]
Can you provide a screen shot showing the error you're getting? Cloudflare's upload limit is 100mb and there shouldn't be any difference between admin UI and frontend.

**Reply by eevleevs** [reply]
I can no longer reproduce it, maybe it was a network issue. Will update if I see it again. Thanks.

**Reply by eevleevs** [reply]
By the way, when you mentioned the 100 MB limit, how do we upload a bigger backup?

**Reply by benallfree** [reply]
FTP
